specs/radiogroup.md

```diff
@@ -0,0 +1,53 @@
+# RadioGroup Component Specifications
+
+Support a similar to native radiogroup implementations. Please see SelectionList implementation to view the new pattern.
+
+## Properties
+
+| Name | Type | Default | Required | Description |
+| -- | -- | -- | -- | -- |
+| value | ?R&D? | false | no | The value chosen in the checkbox |
+| onChange | ?R&D? | NOOP | no | Triggered by changing a radio button state |
+| children | Array<Node> | null | no | children | radio buttons and/or other nodes which will just be rendered |
+| dataSource | Array\<Object> | [] | no | There are a few options accepted as a datasource (see below for explanation) |
+| dataSchema | object | { id: 'id', displayText: 'displayText' } | no | Maps the object properties to the relevant properties required by the ItemRenderer |
```

Comment:
It's possible. This object is used to map the information coming from the datasource to the itemrenderer (default or custom). You can decide on anything you want.